web-console/src/views/load-data-view/ingestion-spec-completions.ts

```diff
@@ -0,0 +1,904 @@
+      { value: 'metricsSpec', documentation: 'Metrics and aggregators configuration' },
+      { value: 'granularitySpec', documentation: 'Granularity and rollup configuration' },
+      { value: 'transformSpec', documentation: 'Transform and filter configuration' },
+    ],
+  },
+  // timestampSpec object properties
+  {
+    path: '$.spec.dataSchema.timestampSpec',
+    isObject: true,
+    completions: [
+      { value: 'column', documentation: 'Input column containing the timestamp' },
+      { value: 'format', documentation: 'Format of the timestamp' },
+      { value: 'missingValue', documentation: 'Default timestamp for missing values' },
+    ],
+  },
+  // timestampSpec.format values
+  {
+    path: '$.spec.dataSchema.timestampSpec.format',
+    completions: [
+      { value: 'auto', documentation: 'Automatically detect ISO or millis format (default)' },
+      { value: 'iso', documentation: 'ISO8601 format with T separator' },
+      { value: 'posix', documentation: 'Seconds since epoch' },
+      { value: 'millis', documentation: 'Milliseconds since epoch' },
+      { value: 'micro', documentation: 'Microseconds since epoch' },
+      { value: 'nano', documentation: 'Nanoseconds since epoch' },
+      { value: 'yyyy-MM-dd HH:mm:ss', documentation: 'Custom Joda format' },
+      { value: 'yyyy-MM-dd', documentation: 'Date only format' },
+    ],
+  },
+  // timestampSpec.column common values
+  {
+    path: '$.spec.dataSchema.timestampSpec.column',
+    completions: [
+      { value: 'timestamp', documentation: 'Default timestamp column name' },
+      { value: '__time', documentation: 'Druid internal timestamp column' },
+      { value: 'time', documentation: 'Common timestamp column name' },
+      { value: 'event_time', documentation: 'Common event time column name' },
+      { value: 'created_at', documentation: 'Common creation time column name' },
+    ],
+  },
+  // dimensionsSpec object properties
+  {
+    path: '$.spec.dataSchema.dimensionsSpec',
+    isObject: true,
+    completions: [
+      { value: 'dimensions', documentation: 'List of dimension specifications' },
+      { value: 'dimensionExclusions', documentation: 'Dimensions to exclude from ingestion' },
+      { value: 'spatialDimensions', documentation: 'Spatial dimension specifications' },
+      { value: 'includeAllDimensions', documentation: 'Include all discovered dimensions' },
+      { value: 'useSchemaDiscovery', documentation: 'Enable automatic schema discovery' },
+      {
+        value: 'forceSegmentSortByTime',
+        documentation: 'Force segments to be sorted by time first',
+      },
+    ],
+  },
+  // useSchemaDiscovery values
+  {
+    path: '$.spec.dataSchema.dimensionsSpec.useSchemaDiscovery',
+    completions: [
+      { value: 'true', documentation: 'Enable automatic type detection and schema discovery' },
+      { value: 'false', documentation: 'Use manual dimension specification (default)' },
```

Comment:
this is perhaps misleading since you can still discover dimensions by leaving the list empty OR by specifying `includeAllDimension` as true.. though maybe that makes it even more confusing..